scheme/parsec-tpm/evidence_handler.go

```diff
@@ -104,3 +225,185 @@ func parsecTpmLookupKey(scope, tenantID, class, instance string) string {
 
 	return u.String()
 }
+
+func evidenceToMap(e parsectpm.Evidence) (map[string]interface{}, error) {
+	data, err := e.ToJSON()
+	if err != nil {
+		return nil, err
+	}
+
+	var out map[string]interface{}
+	err = json.Unmarshal(data, &out)
+
+	return out, err
+}
+
+func populateAttestationResult(
```

Comment:
General comment for this function: I thought all syntactic errors should be marked as `handler.BadEvidence`?  @setrofim could you please shed some light?

Reply:
Yes, good point. Given that this is internal, the easiest thing to do is to wrap the return from this in `handler.BadEvidence` on line 178 above.